app/konnect/dev-portal/developers/dev-reg.md

```diff
@@ -1,51 +1,27 @@
+Everyone accessing a {{site.konnect_short_name}} Dev Portal needs to register for that specific Dev Portal as a Developer, including Dev Portal admins. This means that admins will have credentials for both [Konnect](https://konnect.konghq.com/) and the Dev Portal. 
 
-Access the Dev Portal for any published Service through the URL
-displayed on the **Dev Portal** page. The format looks something like this:
+All Developer registrations are either automatically or manually approved by {{site.konnect_short_name}} admins.
 
-```
-<org-name>.portal.konnect.konghq.com/portal/published-services
-```
+1. As an admin, log in to [konnect.konghq.com](https://konnect.konghq.com/). 
 
-1. Navigate to the {{site.konnect_short_name}} Dev Portal page.
+2. Use the left-side menu to navigate to **Dev Portal**, then **Published Services**. 
 
-   1. From the **Services** page, click the **Dev Portal** menu.
-   2. In **Published Services**, click the **Portal URL** link.
+3. Under the title heading **Published Services**, click the **Portal URL** link.
 
-   ![Konnect Portal URL](/assets/images/docs/konnect/konnect-pub-services-portal-url.png)
+4. The **Portal URL** leads to a login page. 
 
-2. In the Dev Portal, click **Sign up**. The {{site.konnect_short_name}} Developer dialog appears.
+5. At the bottom of the login form, click **Sign Up**. 
 
-   ![Konnect Developer Registration](/assets/images/docs/konnect/konnect-dev-sign-up.png)
+6. Fill out the registration form and click **Create Account**. 
 
-3. Complete the {{site.konnect_short_name}} Developer dialog:
+7. Check your email for registration confirmation. 
 
-   1. Enter your **First Name** and **Last Name**.
+8. Once your email is confirmed, you _will not be able_ to log in until an admin approves your request. However, if the admin has set the Konnect account to [auto approve](/konnect/dev-portal/administrators/auto-approve-devs-apps) developer registrations, you will be able to login immediately. 
 
-   2. Enter your **Email** address.
-
-   3. Enter a **Password**.
-
-      The password must be a minimum of 8 characters long and contain at least two of the
-      following: a lowercase letter, an uppercase letter, or a special
-      character.
-
-   4. Click **Create Account**.
-
-   5. Check your email for registration confirmation. In the confirm your address email, click the confirmation link.
-
-   6. A message indicates your email has been confirmed, and your account is awaiting approval
-      if [auto approve](/konnect/dev-portal/administrators/auto-approve-devs-apps) is not enabled.
-
-6. After your request has been [approved by an admin](/konnect/dev-portal/administrators/manage-devs) or automatically approved,
-   click **Login**.
-
-7. Enter your credentials and log in. If you experience any access issues, contact your {{site.konnect_short_name}} admin.
+9. Once approved, log in using your Portal credentials. Contact your {{site.konnect_short_name}} admin if you have login issues.
```

Comment:
```suggestion
9. Log in using your Portal credentials. Contact your {{site.konnect_short_name}} admin if you have login issues.
```
Unfortunately, currently they have no way of knowing when they're been approved, right :( It might be good to move all of step 8 into this step: try to log in, oh you can't? that means auto-approve isn't on and you have to wait for your admin.